gitops-deployment/wsa/templates/subscription.yaml

```
@@ -1,4 +1,3 @@
-{{- if .Values.ownNamespace}}
```

Should this be something like 

```suggestion
{{- if .Values.wsaOperatorNamespace == .Values.wsaInstanceNamespace}}
```